src/components/Dashboard/TodoList.vue

```diff
@@ -260,22 +260,27 @@
                   v-else-if="EnumUtilities.isStatusChangeRequested(item)"
                   class="todo-subtitle pt-1"
                 >
-                  <span class="orange--text text--darken-2">CHANGE REQUESTED</span>
+                  <span class="orange--text text--darken-2 font-weight-bold">CHANGE REQUESTED</span>
                   <span class="vert-pipe" />
-                  <span>PAID (filed by {{ item.submitter }} on <DateTooltip :date="item.submittedDate" />)</span>
+                  <span>Submitted by {{ item.submitter }} on <DateTooltip :date="item.submittedDate" />)</span>
                 </div>
```

Comment:
The UI design also shows the NR info here.

![image](https://github.com/user-attachments/assets/95e9a207-6e0e-46d7-b5db-a5847fe1635c)

Reply:
Updated the subtitle: see
https://business-filings-dev--pr-697-l4j2px9g.web.app/Tf6qM3eDoF

Reply:
Looks great!